test/logger.browser.test.js

```diff
@@ -18,7 +18,7 @@ it('prints dispatches', function () {
   var store = createStore([counter, logger])
   store.dispatch('inc', 2)
   expect(console.log.mock.calls).toEqual([
-    ['%caction %c@init', STYLE, BOLD],
+    ['%caction %c@init', STYLE, BOLD, ''],
```

Comment:
this is side effect of https://github.com/storeon/storeon/pull/67/files#diff-f1e2f1f81deedeb95c3c98ed6a7a8649R8
it's better than before so I decided to leave it